templates/verification-email.html

```diff
@@ -0,0 +1,59 @@
+<html>
+  <head>
+    <style type="text/css">
+      @import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@500&family=Poppins:wght@700&display=swap');
+      .main {
+        padding-top: 16px;
+        text-align: center;
+        width: 375px;
+        margin: auto;
+      }
+      h1, .confirm-btn {
+        font-family: 'Poppins', sans-serif;
+        font-weight: bold;
+      }
+      h1 {
+        color: rgba(0, 0, 0, 0.85);
+        font-size: 22px;
+        line-height: 1.17;
+        margin: 22px 50px 42px;
+      }
+      p {
+        font-family: 'DM Sans', sans-serif;
```

Comment:
this one probably also needs to be loaded? I remember there are issues with this font in the main repo too, might be useful to add some more similar fallback fonts otherwise